README.md

```diff
@@ -306,48 +306,45 @@ Router.onRouteChangeError = (err, url) => {
   <ul><li><a href="./examples/with-prefetching">Prefetching</a></li></ul>
 </details></p>
 
-Next.js exposes a module that configures a `ServiceWorker` automatically to prefetch pages: `next/prefetch`.
+Next.js has an API which allows you to prefetch pages.
 
 Since Next.js server-renders your pages, this allows all the future interaction paths of your app to be instant. Effectively Next.js gives you the great initial download performance of a _website_, with the ahead-of-time download capabilities of an _app_. [Read more](https://zeit.co/blog/next#anticipation-is-the-key-to-performance).
 
+> With prefetching Next.js only download JS code. When the page is getting rendered, you may need to wait for the data.
+
 #### With `<Link>`
 
-You can substitute your usage of `<Link>` with the default export of `next/prefetch`. For example:
+You can add `prefetch` prop to any `<Link>` and Next.js will prefetch those pages in the background.
```

Comment:
Do we need to do that? I think it's better users turn it on.
So, we have no surprises.